addons/website_event_track/models/event_track.py

```diff
@@ -86,7 +89,9 @@ def _get_default_stage_id(self):
         ('0', 'Low'), ('1', 'Medium'),
         ('2', 'High'), ('3', 'Highest')],
         'Priority', required=True, default='1')
-    image = fields.Image("Image", max_width=128, max_height=128)
+    image = fields.Image(
```

Comment:
Hello,

Actually computing too much fields in a single method with editable stored fields is not a good idea after recent feedbacks. Indeed if the user gives one of the field, the other one are not computed as the ORM considers the compute method as "already achieved".

I think we can add compute_partner_biography and compute_image methods to avoid side effects like this.